Source/WebCore/rendering/RenderBox.h

```diff
@@ -134,6 +134,7 @@ class RenderBox : public RenderBoxModelObject {
     void addFocusRingRects(Vector<LayoutRect>&, const LayoutPoint& additionalOffset, const RenderLayerModelObject* paintContainer = nullptr) const override;
     
     FloatRect repaintRectInLocalCoordinates() const override { return borderBoxRect(); }
+    FloatRect approximateRepaintRectInLocalCoordinates() const override { return borderBoxRect(); }
```

Comment:
Not calling `repaintRectInLocalCoordinates` to avoid virtual function dispatch twice.